### PR TITLE
NO-JIRA: Clean carry patches

### PR DESCRIPTION
**Pull request by soltysh**

### CODE REVIEW

File: openshift-hack/e2e/namespace.go [hunk 1]
```diff
@@ -33,6 +33,9 @@ func CreateTestingNS(ctx context.Context, baseName string, c kclientset.Interfac
 			labels = map[string]string{}
 		}
 		labels["security.openshift.io/disable-securitycontextconstraints"] = "true"
+		// turn off the OpenShift label syncer so that it does not attempt to sync
```
Comment (stlaz) [inline openshift-hack/e2e/namespace.go:36]: when is "isKubeNamespace" false?

Comment (soltysh) [reply openshift-hack/e2e/namespace.go:36]: In origin we have it set here: https://github.com/openshift/origin/blob/master/test/extended/util/test_setup.go#L89-L91, in k8s it's always true https://github.com/openshift/kubernetes/blob/master/openshift-hack/cmd/k8s-tests/provider.go#L103-L105

**Review by stlaz — CHANGES_REQUESTED** [review]

**Review by stlaz — CHANGES_REQUESTED** [review]
LGTM auth-wise.
Leaving review to @dinhxuanvu for the API things.

**Comment by soltysh** [comment]
/remove-label backports/unvalidated-commits
/remove-label backports/validated-commits
/hold
to make sure we don't break origin <todo> link origin PR here

**Review by dinhxuanvu — COMMENTED** [review]
/lgtm

**Comment by soltysh** [comment]
We have sufficient proof in https://github.com/openshift/origin/pull/28588
/hold cancel

**Pull request merged**